asserts/authority_delegation_test.go

```diff
@@ -240,3 +268,17 @@ func (s *authorityDelegationSuite) TestDecodeInvalid(c *C) {
 		c.Check(err, ErrorMatches, authDelegErrPrefix+test.expectedErr)
 	}
 }
+
+func (s *authorityDelegationSuite) TestDecodeDeviceScope(c *C) {
+	// XXX: for now we fail on device scope constraints
+	// to avoid misinterpreting assertions until it is properly implemented
+	encoded := s.validEncoded
+	sinceFrag := "    since: 2022-01-12T00:00:00.0Z\n"
+	for _, deviceScoping := range []string{
+		"on-store: store1",
```

Comment:
Did you mean to cover "on-brand" and "on-model" here as well?

Reply:
yes